dgc/DGCertSupportedAlgorithm.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace DGC
+﻿namespace DCC
```

Comment:
How are you handling multiple schema versions?

Reply:
currently, I don't see any need for it because a v1.3 is backwards compatible with v1.0